hubmap_api_py_client/external.py

```diff
@@ -163,12 +163,83 @@ def _add_method(output_type, ResultsSetSubclass, doc):
 
 
 for output_type, doc in {
-    'cell': 'TODO: document cell',
-    'organ': 'TODO: document organ',
-    'gene': 'TODO: document gene',
-    'cluster': 'TODO: document cluster',
-    'dataset': 'TODO: document dataset',
-    'protein': 'TODO: document protein'
+    'cell':
+    '''
+    Select a set of cells. If no params are provided, selects the set of all cells.
+    Otherwise, selects a set of cells filtered based on parameters supplied.
+        where->str The type of entity being supplied as input to query, must be one of
+            ["cell", "gene", "protein", "organ", "dataset"]
+        has->List[str] A list of entities or expressions supplied as input to the query
```

Comment:
```suggestion
        has->List[str] A list of UUIDs, or of expressions like "GENE > 0.2".
```

Reply:
While the identifiers are universally unique, I feel like the use of UUID, especially in a HuBMAP context has a specific connotation that's not intended here.  Maybe entity identifiers instead?

Reply:
Sure: or just plain "IDs" would be fine, too. I think of entities as things that are referenced by ID, so I was just a little confused by that part in the original phrasing.